install/docker-compose/docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: '2'
+
+services:
+  mongo:
+    image: mongo:3.4.23
+    container_name: microcks-db
+    volumes:
+      - "~/tmp/microcks-data:/data/db"
+
+  keycloak:
+    image: jboss/keycloak:10.0.1
+    container_name: microcks-sso
```

Comment:
Could you change to the old name that is `microcks-keycloak`? That way we know it's the container attached to the `keycloak` alias.